### ISSUE TITLE
Need help with getting bboxes.

### ISSUE THREAD

**Issue by MrHorsemann7** [eriklindernoren/PyTorch-YOLOv3#803]
I am trying to use the model on the Django web app. I was copied the flow of detect_image function from pytorcyolo/detect.py. But its returning empty list every time ı tried. Can someone please help me?

CODE:
`
class VideoCameraDetect:
    def __init__(self, model_path, mp4_path=None):
        if mp4_path:
            self.video = cv2.VideoCapture(mp4_path)
        else:
            self.video = cv2.VideoCapture(0)
        
        path = model_path.split("/")
        path =  [i for i in path if ".pth" in i]
        path = path[0].replace("_model.pth",".names")
        
        self.idx = 0
        self.pred = "test"
        self.coord = [30,35,100,100]
        
        n_classes, self.labels = get_labels(path)
        self.model = load_model("D:/ide/Saves/RedHorse/env/HorseNet/NN/YOLO/config/yolov3-tiny.cfg",weights_path=model_path)
        self.model_path = model_path
        summary(self.model, input_size=(3, 416, 416))
        
        self.grabbed, self.frame = self.video.read()
        threading.Thread(target=self.update, args=()).start()

    def __delete__(self):
        self.video.release()
        
    def get_frame(self):
        frame = self.frame
        image = cv2.resize(frame,(416,416))
        #image = cv2.cvtColor(image, cv2.COLOR_RGB2BGR)
        if self.idx % 10 == 0:
            image = transforms.Compose([
                DEFAULT_TRANSFORMS,
                Resize(416)])(
                    (image, np.zeros((1, 5))))[0].unsqueeze(0)
            if torch.cuda.is_available():
                image = image.to("cuda")
            with torch.no_grad():
                print("1",image.shape)
                detections = self.model(image)
                print("2",detections[0].shape)
                detections = non_max_suppression(detections)
                detections = rescale_boxes(detections[0], 416, image.shape[:2])
                print("3",detections)
                self.coord = detections.numpy()
`
OUTPUT:
1 torch.Size([1, 3, 416, 416])
2 torch.Size([1, 3, 13, 13, 85])
3 tensor([], size=(0, 6))
